test/simulator/timestamp/src/timestamp_manager.cpp

```diff
@@ -29,8 +29,8 @@
 #include <iostream>
 #include <string>
 
-#include "timestamp_manager.h"
 #include "connection_simulator.h"
+#include "timestamp_manager.h"
```

Comment:
np: [MongoDB guidelines](https://github.com/mongodb/mongo/wiki/Server-Code-Style#for-cpp-files) state that the header of a class always comes first.

Reply:
Thanks.

Reply:
Updated in [81a105e](https://github.com/wiredtiger/wiredtiger/pull/8116/commits/81a105ec3ac049c559271087f59c6b328923d2f8)